examples/denoise_covar2d.py

```diff
@@ -126,17 +132,17 @@
 # estimates. For the covariance estimation, the additional information of
 # the estimated mean and the variance of the noise are needed. Again, the
 # covariance matrix estimate is provided in block diagonal form.
-cov2dctf = Cov2DCTF(sim, ffbbasis)
-mean_coeff_est = cov2dctf.get_mean_ctf(coeff_noise, h_ctf_fb, h_idx)
-covar_coeff_est = cov2dctf.get_covar_ctf(coeff_noise, h_ctf_fb, h_idx, mean_coeff_est, noise_var=noise_var)
+mean_coeff_est = cov2d.get_mean_ctf(coeff_noise, h_ctf_fb, h_idx)
```

Comment:
As discussed, we shouldn't have separate functions (at least exposed publicly) for with and without CTF. The CTF case can be reduced to the non-CTF case by assigning all images to the same filter and making that filter equal to the identity.

Reply:
I will try to combine the non-CTF and CTF methods in the same class.

Reply:
I merged them by checking the CTF information exists or not. It looks that we don't need to apply the identity filter at all. The update is done in commit 454d85a.

Reply:
Ok.